alluxiofs/core.py

```diff
@@ -120,6 +116,17 @@ def __init__(
             if preload_path is not None:
                 self.alluxio.load(preload_path)
 
+        if "log_level" in test_options:
```

Comment:
think u should include in the options

Reply:
alluxiofs is an inner Python module
For each Python process, the end launch program should define to log level, and all the inner Python module inherit the logging configuration.
E.g. in our benchmark entry point, we should define the log level and log file there

Reply:
no I meant to say why not use the dict "options" ? its also user defined at time of instantiating alluxiofs instance right?